tests/components/samsungtv/test_config_flow.py

```diff
@@ -1255,7 +1253,6 @@ async def test_autodetect_legacy(hass: HomeAssistant) -> None:
     )
     assert result["type"] is FlowResultType.CREATE_ENTRY
     assert result["data"][CONF_METHOD] == "legacy"
-    assert result["data"][CONF_NAME] == "fake_name"
```

Comment:
I have reviewed this more thoroughly, and I think we should keep the statement, possibly with a comment:
```python
    # friendly name cannot be detected for legacy TV 
    assert result["data"][CONF_NAME] is None
```

Reply:
This was my initial intention, even if I didn't think that a comment was "needed" ;-)